### PR TITLE
Accessibility evaluation

### PR DESCRIPTION
**Pull request by juanmanuelbyc**
We are going through these aspects:

**Page titles - Changed the title to "Juan Diaz Portfolio"**
https://github.com/juanmanuelbyc/Microverse-Student-Project-1/pull/8/files#diff-0eb547304658805aad788d320f10bf1f292797b5e6d745a3bf617584da017051R9

**Image text alternatives - updated**
https://github.com/juanmanuelbyc/Microverse-Student-Project-1/pull/8/files#diff-0eb547304658805aad788d320f10bf1f292797b5e6d745a3bf617584da017051R40-R44

https://github.com/juanmanuelbyc/Microverse-Student-Project-1/pull/8/files#diff-0eb547304658805aad788d320f10bf1f292797b5e6d745a3bf617584da017051R175-R179

**Text heading - Fixed heading hierarchy for the main heading at the beginning of the page**
https://github.com/juanmanuelbyc/Microverse-Student-Project-1/pull/8/files#diff-0eb547304658805aad788d320f10bf1f292797b5e6d745a3bf617584da017051R30

**Color contrast - fixed several text colors** (also there's others that won't pass contrast text. But changing them would make them way different than Figma design)

https://github.com/juanmanuelbyc/Microverse-Student-Project-1/pull/8/files#diff-f6e4f9cac7473cf1fe513beba97669f195a6e1872bbd857c713d41e0f4b5d289R46

https://github.com/juanmanuelbyc/Microverse-Student-Project-1/pull/8/files#diff-f6e4f9cac7473cf1fe513beba97669f195a6e1872bbd857c713d41e0f4b5d289R122

https://github.com/juanmanuelbyc/Microverse-Student-Project-1/pull/8/files#diff-f6e4f9cac7473cf1fe513beba97669f195a6e1872bbd857c713d41e0f4b5d289R256

https://github.com/juanmanuelbyc/Microverse-Student-Project-1/pull/8/files#diff-f6e4f9cac7473cf1fe513beba97669f195a6e1872bbd857c713d41e0f4b5d289R266

https://github.com/juanmanuelbyc/Microverse-Student-Project-1/pull/8/files#diff-f6e4f9cac7473cf1fe513beba97669f195a6e1872bbd857c713d41e0f4b5d289R266

**Resize  - no issues found**
**Interaction - no issues found**
**Moving content - no issues found**
**Multimedia - no issues found**
**The basic structure of the page - no issues found**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -29,16 +30,18 @@
           <h2 class="primary-text">I'm Juan Diaz Glad to see you!</h2>
```
Comment (omarsalem7) [inline index.html]: - [x] Consider the heading hierarchy is meaningful. Ideally, the page starts with an "h1" , so this should be an `h1` instead of `h2`

**Review by omarsalem7 — CHANGES_REQUESTED** [review]
Hi @juanmanuelbyc 😊, 

Good job so far! 👏
You have done well reaching this point of the milestone, but there are some issues that you still need to work on to go to the next project but you are almost there! 

you-are-near

## Highlights
- Descriptive pull request ✔️
- No linter errors ✔️
- The website resizes correctly ✔️

## Required Changes ♻️
_Check the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (juanmanuelbyc) [reply index.html]: Done! thanks for the feedback!

**Review by sumairq — APPROVED** [review]
### Hi @juanmanuelbyc @kidde60  , 



Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
✔️ Accessibility requirements are properly met
✔️ Good commit history
✔️Linters are passing
✔️Very professional README
✔️Correct Github flow
✔️Descriptive PR
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**